MetadataExtractor/Formats/QuickTime/QuickTimeReaderExtensions.cs

```diff
@@ -9,12 +9,11 @@ public static class QuickTimeReaderExtensions
     {
         public static string Get4ccString(this SequentialReader reader)
         {
-            var sb = new StringBuilder(4);
-            sb.Append((char)reader.GetByte());
-            sb.Append((char)reader.GetByte());
-            sb.Append((char)reader.GetByte());
-            sb.Append((char)reader.GetByte());
-            return sb.ToString();
+            Span<byte> bytes = stackalloc byte[4];
+
+            reader.GetBytes(bytes);
+
+            return new string([(char)bytes[0], (char)bytes[1], (char)bytes[2], (char)bytes[3]]);
```

Comment:
.NETStandard2.1 has a System.ReadOnlySpan<char> overload that stack allocates. However, agree on updating to ASCII.GetString to avoid the char[] allocation on other targets.

The callers here expect this function to produce a string, so not much else to optimize here.

<img width="841" alt="Screenshot 2024-01-29 at 6 55 06 PM" src="https://github.com/drewnoakes/metadata-extractor-dotnet/assets/7537/8b9706c5-01ed-424d-979f-61b5aecf567b">

Reply:
Turns out my advice to use Encoding.ASCII wasn't that great ;)